graphql/fetchUserRole.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from "@apollo/client";
+
+export const FETCH_USER_ROLE = gql`
+query FetchUserRole($_id: String = "") {
+    users(where: {id: {_eq: $_id}}) {
+      userRole {
+        value
+      }
+    }
+  }
+`;
+
+export type FetchUserRoleData = {
+    users: UserRoleData;
+}
+
+
+export type UserRoleData = {
+    userRole: UserRole;
```

Comment:
instead of defining a separate UserRole type, pretty sure you can just declare userRole only takes two specific strings "student" or "coach" i.e userRole: "student" | "coach"

Reply:
makes sense.  I changed it.  I thought we needed to type out the data for the fetchUserRole query we were implementing on the layout.tsx.